nemo/core/classes/common.py

```diff
@@ -236,34 +241,54 @@ def to_config_file(self, path2yaml_file: str):
             raise NotImplementedError()
 
 
+PretrainedModelInfo = namedtuple("PretrainedModelInfo", ("pretrained_model_name", "description", "location"),)
+
+
 class Model(Typing, Serialization, FileIO):
     """
     Abstract class offering interface which should be implemented by all NeMo models.
     """
 
     @classmethod
     @abstractmethod
-    def list_available_models(cls) -> Optional[Dict[str, str]]:
+    def list_available_models(cls) -> Optional[PretrainedModelInfo]:
         """
         Should list all pre-trained models available via NVIDIA NGC cloud
 
         Returns:
-            A dictionary of NeMo model key name -> NGC wget URI
+            A list of PretrainedModelInfo entries
         """
         pass
 
     @classmethod
-    @abstractmethod
-    def from_pretrained(cls, name: str):
+    def from_pretrained(cls, model_name: str, refresh_cache: bool = False):
         """
         Instantiates an instance of NeMo from NVIDIA NGC cloud
         Args:
-            name: string key which will be used to find the module. Could be path to local .nemo file.
-
+            model_name: string key which will be used to find the module. Could be path to local .nemo file.
+            refresh_cache: If set to True, then when fetching from clould, this will re-fetch the file
+                from clould even if it is  already found in a cache locally.
```

Comment:
fetching from "cloud", below as well. Extra space in between "it is  already found"

Reply:
fixed